core/src/test/scala/org/broadinstitute/dsde/rawls/user/UserServiceSpec.scala

```diff
@@ -88,11 +88,13 @@ class UserServiceSpec extends AnyFlatSpecLike with TestDriverComponent with Mock
       runAndWait(rawlsBillingProjectQuery.create(project))
 
       val mockGcsDAO = mock[GoogleServicesDAO](RETURNS_SMART_NULLS)
-      val googleProjectNumber = 42L
-      when(mockGcsDAO.getGoogleProject(project.googleProjectId)).thenReturn(Future.successful(new Project().setProjectNumber(googleProjectNumber)))
+      val googleProjectNumber = GoogleProjectNumber("42")
+      val googleProject = new Project().setProjectNumber(googleProjectNumber.value.toLong)
+      when(mockGcsDAO.getGoogleProject(project.googleProjectId)).thenReturn(Future.successful(googleProject))
       val folderId = "folders/1234567"
       when(mockGcsDAO.getFolderId(defaultServicePerimeterName.value.split("/").last)).thenReturn(Future.successful(Option(folderId)))
       when(mockGcsDAO.addProjectToFolder(project.googleProjectId, folderId)).thenReturn(Future.successful(()))
+      when(mockGcsDAO.getGoogleProjectNumberFromGoogleProject(googleProject)).thenReturn(googleProjectNumber)
```

Comment:
I'm not sure you need to mock this?  I think that function should be self contained and should just work now?